pyproject.toml

```diff
@@ -51,6 +51,10 @@ pii = [
     # This may be required for the larger models due to Databricks connect memory limitations.
     # The models cannot be delcared as dependency here buecase PyPI does not support URL-based dependencies which would prevent releases.
 ]
+llm = [ # LLM assisted features
+    "dspy-ai~=3.0.3",
```

Comment:
These must be added to dependencies as well so that the libs are available for testing
```
[tool.hatch.envs.default]
dependencies = [
  "dspy-ai~=3.0.3",
 "databricks-langchain~=0.8.0",
....
```